packages/astro/src/@types/astro.ts

```diff
@@ -724,6 +724,15 @@ export interface AstroUserConfig {
 		 * @description
 		 * Enable Astro's pre-v1.0 support for components and JSX expressions in `.md` Markdown files.
 		 * In Astro `1.0.0-rc`, this original behavior was removed as the default, in favor of our new [MDX integration](/en/guides/integrations-guide/mdx/).
+		 * 
+		 * To enable this behavior, set `legacy.astroFlavoredMarkdown` to `true` in your [astro.config.mjs]{/en/guides/configuring-astro/} configuration file.
+		 * 
+		 * {
+		 *  legacy: {
+		 *    	//example: Add support for legacy .md files
+		 * 			astroFlavoredMarkdown: true,
+		 * 		},
+		 * 	}
```

Comment:
```suggestion
		 * To enable this behavior, set `legacy.astroFlavoredMarkdown` to `true` in your [astro.config.mjs]{/en/guides/configuring-astro/} configuration file.
		 * 
		 * ```js
		 * {
		 *   legacy: {
		 *    	// Example: Add support for legacy Markdown features
		 * 			astroFlavoredMarkdown: true,
		 *   },
		 * }
		 * ```
```